x/ecocredit/client/tx.go

```diff
@@ -189,8 +189,8 @@ func txCancel() *cobra.Command {
 		Long: `Cancels a specified amount of credits from the account of the transaction author (--from)
 
 Parameters:
-  credits:  comma-separated list of credits in the form <amount>:<batch-denom>
-            eg: 10:ABC/123,0.1:XYZ/456`,
+  credits:  comma-separated list of credits in the form <amount><batch-denom>
+            eg: 10eco:ABC:123,0.1eco:XYZ:456`,
```

Comment:
Looks good. What do you think of hyphens for the class and batch separator? I.e.
```suggestion
            eg: 10eco:ABC-123,0.1eco:XYZ-456`,
```

I'm thinking of what these might look like when placed in a URI.

Reply:
I do find the multiple colons a bit odd.

> Also, for IID/DID compatibility we want to switch from : to / separators, i.e. 18AV53K:1b1vrHF.

I don't know what IID/DID is though, so not sure what the requirements for compatibility are. Is it just not having a `/`?